src/components/like-button.vue

```diff
@@ -60,4 +60,19 @@ export default defineComponent({
     IonLabel, IonIcon,
   },
 });
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+
+ion-chip span{
+  margin-top:2px;
+  padding-right: 6px !important; 
```

Comment:
Any way we can get around using `!important`? in my experience this always bites one in the butt later.

Reply:
This is for make sure no other same class overrides its property padding-right. As same classes used onside other ion-chips.